### PR TITLE
freetds: upgrade 1.1.20 -> 1.4.10

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Upgrade 1.1.20 -> 1.3.20

###### Change Log  

- Upgrade 1.1.20 -> 1.3.20

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

Pipeline build id: 
- [PR-7094](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=479824&view=logs&j=21e55549-fa44-57d8-4936-64132e3ceec0&t=8e6f9516-5704-5480-d4b1-637f5c23550a) ❌ 
- [PR-7094](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=481552&view=logs&j=7909073b-e471-53c1-2c44-627e2efc0564&t=fbeef549-39ab-510c-faa3-78579a516b53) ✅ 

local build:
image

### CODE REVIEW

File: SPECS/freetds/freetds.spec [hunk 1]
```diff
@@ -1,8 +1,8 @@
 %define bits	%{?__isa_bits:%{__isa_bits}}%{!?__isa_bits:32}
 Summary:        Implementation of the TDS (Tabular DataStream) protocol
 Name:           freetds
-Version:        1.1.20
-Release:        5%{?dist}
+Version:        1.3.20
```
Comment (christopherco) [inline SPECS/freetds/freetds.spec:4]: Curious if we can move forward to v1.4.10 which was released recently

Comment (mfrw) [reply SPECS/freetds/freetds.spec:4]: Ack. Sure :)

Comment (mfrw) [reply SPECS/freetds/freetds.spec]: Done

File: SPECS/freetds/freetds.signatures.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "freetds-1.1.20.tar.bz2": "fc549b945208c42264f8e31ba6a8c94f8a9b441802f3b952d11445035daac41a",
+  "freetds-1.4.10.tar.bz2": "c7eaf226bdcb1cdc1b221696532ccd25f4e4e7754265a29777a3400084bde698",
   "freetds-tds_sysdep_public.h": "8478bd6bd98ba4bfaf5920aae639b155d7fe2eec676bdbbd9d01e1358f295a59"
  }
 }
```
Comment (mfrw) [inline SPECS/freetds/freetds.signatures.json:6]:
```suggestion
}

```

**Review by kanikanema — APPROVED** [review]
LGTM

**Pull request merged**